src/renderer/pages/project/script_editor/reference.vue

```
@@ -74,6 +74,9 @@
 
 import ReferenceSelector from "./reference_selector.vue";
 
+import { getConcurrentTaskStatusMessageComponent } from "../concurrent_task_status_message";
+import { notifySuccess, notifyProgress } from "@/lib/notification";
```

ESLint

[error] 'notifySuccess' is defined but never used. Allowed unused vars must match /^__/u.

(@typescript-eslint/no-unused-vars)